backend/handlers/base_handler.py

```diff
@@ -14,7 +14,7 @@ class BaseHandler(webapp2.RequestHandler):
     def decorateHeaders(self):
         """Decorates headers for the current request."""
         self.response.headers['Access-Control-Allow-Origin'] = '*'
-        self.response.headers['Access-Control-Allow-Headers'] = 'X-API-KEY, Origin, X-Requested-With, Content-Type, Accept, Access-Control-Request-Method, Authorization'
+        self.response.headers['Access-Control-Allow-Headers'] = 'X-API-KEY, Origin, X-Requested-With, Content-Type, Accept, Access-Control-Request-Method, Authorization, Institution-Authorization'
```

Comment:
Erro ao tentar postar algo:
File "/home/mayza/workspace/web/backend/handlers/post_collection_handler.py", line 43, in post
    post_data = data['post']
KeyError: 'post'

Reply:
:+1: